query-engine/driver-adapters/executor/src/bench.ts

```diff
@@ -57,23 +64,36 @@ async function recordQueries(
   datamodel: string,
   prismaQueries: any
 ): Promise<void> {
-  const qe = await initQeWasmBaseLine(adapter, datamodel);
-  await qe.connect("");
+  // Different engines might have made different SQL queries to complete the same Prisma Query,
+  // so we record the results of all engines and assert that they are the same.
```

Comment:
Where are we asserting that the result of each Query Engine is the same?

Reply:
Leftover, removed this as it increased notably CPU time and skewed results